src/constants.js

```diff
@@ -19,5 +21,20 @@ module.exports.ERROR_MESSAGE_TRANSACTION_NO_HASH =
 module.exports.ERROR_MESSAGE_ADDRESS_NO_ADDRESS = 'No address supplied'
 module.exports.ERROR_MESSAGE_CONTRACT_NO_ADDRESS =
   'No contract address supplied'
-module.exports.ERROR_MESSAGE_BLOCK_NO_NUMBER = 'No block number supplied'
+module.exports.ERROR_MESSAGE_BLOCK_NO_ID = 'No block number or hash supplied'
 module.exports.ERROR_MESSAGE_SIGNATURE_NO_HASH = 'No signature hash supplied'
+
+/* Ethereum based methods */
+module.exports.ETH_METHODS = {
+  getBlockNumber: 'block',
+  getBlock: 'block',
+  getBlockTransactionCount: 'block',
+  getUncle: 'block',
```

Comment:
@taylorjdawson `getBalance` method needs to be here, right?

Reply:
added!